### PR TITLE
Added Instagram Login Page Clone to Basic Web Development Project section

### LINKED ISSUE #397: Add Instagram App Login Page Clone
### Is your feature request related to a problem? Please describe.

This is a basic web development project which simulates Instagram App Login/Signup Page. 

### Describe the solution you'd like.

An Instagram Login/Sign up Page Clone built using HTML and CSS

### Describe alternatives you've considered.

No alternatives as of now

### Add any other context or screenshots about the feature request here.

https://user-images.githubusercontent.com/80174214/158025772-fabfcede-5aae-43f9-8724-9f7a83955363.mp4

Issue made under GSSOC'22 Contributions

### PR DESCRIPTION
**Pull request by abhilipsasahoo03**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes. 

### Description

A simple Instagram App Official Login Page Clone built using HTML and CSS. [Here](https://www.instagram.com/accounts/login/?) is a reference to the official sign up/login page of Instagram for comparison.

Fixes #397 

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` and link to my code.

### CODE REVIEW

**Review by photon149 — APPROVED** [review]
@abhilipsasahoo03 Good One !
@Kushal997-das @prakhar3062 have a look.

**Review by prakhar3062 — APPROVED** [review]

File: Web Development/README.md [hunk 1]
```diff
@@ -42,6 +42,8 @@
   - [To Do App](https://github.com/Kushal997-das/Project-Guidance/tree/main/Web%20Development/Basic/To%20Do%20App)<br>
 
   - [Tip Calculator](https://github.com/RiyaBhandari-2811/Project-Guidance/tree/TipForm/Web%20Development)<br>
+  
+  - [Instagram Login Page Clone](https://abhilipsasahoo03.github.io/Instagram-Login-Page-Clone/)<br>
```
Comment (Kushal997-das) [inline Web Development/README.md:46]: Change this link to your project location link see above work you can get idea.

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@abhilipsasahoo03 .

**Comment by abhilipsasahoo03** [comment]
Hi @Kushal997-das I've made the appropriate changes please look into it

**Review by Kushal997-das — APPROVED** [review]

**Comment by Kushal997-das** [comment]
@abhilipsasahoo03 Great 💯

**Comment by Kushal997-das** [comment]
@abhilipsasahoo03 We gonna remove the tag , just because you added same project in other repo.

**Pull request merged**